src/PIL/PpmImagePlugin.py

```diff
@@ -116,6 +116,8 @@ def _open(self):
                     break
             elif ix == 2:  # token is maxval
                 maxval = token
+                if maxval >= 65536 or maxval == 0:
+                    raise ValueError("Maxval not greater than 0 and less than 65536")
```

Comment:
This is a little ambiguous, it could be what is wrong or what it should be.

Maybe something along the lines of "Maxval must be greater than 0 and less than 65536" or "Maxval must be between 1-65535"?

(And lowercase maxval even if at start of the sentence, the case is important for a variable.)

Reply:
Ok, updated.